labelbox/data/serialization/ndjson/objects.py

```diff
@@ -211,6 +287,15 @@ def from_common(
         cls, annotation: ObjectAnnotation, data: Union[ImageData, TextData]
     ) -> Union[NDLine, NDPoint, NDPolygon, NDRectangle, NDMask, NDTextEntity]:
         obj = cls.lookup_object(annotation)
+
+        #if it is video segments
+        if (obj == NDSegments):
+            return obj.from_common(
+                annotation,
+                data,
+                feature_schema_id=annotation[0][0].feature_schema_id,
+                extra=annotation[0][0].extra)
```

Comment:
How does this work? annotation is cannot be indexed if it is actually an ObjectAnnotation.

Reply:
Sorry, this is a List[List[VideoObjectAnnotation]]. I have updated the type hint to reflect this